css2/Overview.bs

```diff
@@ -15,6 +15,7 @@ Former Editor: Editor: Bert Bos, W3C, mailto:bert@w3.org, w3cid 3343
 Former Editor: Elika J. Etemad / fantasai, Invited Expert, http://fantasai.inkedblade.net/contact, w3cid 35400
 Former Editor: Ian Hickson, ian@hixie.ch
 Former Editor: H&aring;kon Wium Lie, Opera Software, howcome@opera.com
+WPT Display: inline
```

Comment:
Did you really intend to commit this line? Feels like something you may have used locally and forgot to remove before committing.